### PR TITLE
restore rag application Containerfile & other fixes 

### PR DESCRIPTION
**Pull request by sallyom**
* adds sqlite install back to rag containerfile
* adds data volume mount to rag pod definition

### CODE REVIEW

File: recipes/natural_language_processing/rag/builds/Containerfile [hunk 1]
```diff
@@ -1,4 +1,16 @@
 FROM registry.access.redhat.com/ubi9/python-311:1-52
```
Comment (rhatdan) [inline recipes/natural_language_processing/rag/builds/Containerfile:1]: Why isn't this latest?

File: recipes/natural_language_processing/rag/builds/Containerfile [hunk 2]
```diff
@@ -1,4 +1,16 @@
...
+### Update sqlite for chroma
+USER root
+RUN dnf remove sqlite3 -y
```
Comment (rhatdan) [inline recipes/natural_language_processing/rag/builds/Containerfile:4]: Is the sqlite package in RHEL not new enough?

 rpm -q sqlite
sqlite-3.34.1-7.el9_3.x86_64

File: recipes/natural_language_processing/rag/builds/Containerfile (hunk 1 above)
Comment (sallyom) [reply recipes/natural_language_processing/rag/builds/Containerfile:1]: we've pinned to this everywhere - @lmilbaum are you ok w/ switching to latest everywhere?

File: recipes/natural_language_processing/rag/builds/Containerfile (hunk 2 above)
Comment (sallyom) [reply recipes/natural_language_processing/rag/builds/Containerfile:4]: application errored without this added back, with this error: 
`Your system has an unsupported version of sqlite3. Chroma requires sqlite3 >= 3.35.0`
so i guess we just missed the cut there :)

File: recipes/natural_language_processing/rag/builds/Containerfile (hunk 1 above)
Comment (rhatdan) [reply recipes/natural_language_processing/rag/builds/Containerfile:1]: My fear is that these do not age well.

File: recipes/natural_language_processing/rag/builds/Containerfile (hunk 2 above)
Comment (rhatdan) [reply recipes/natural_language_processing/rag/builds/Containerfile:4]: Ok, hopefully 9.4 has an update.

**Review by MichaelClifford — APPROVED** [review]
lgtm

**Pull request merged**